opentech/apply/projects/templates/application_projects/includes/report_line.html

```diff
@@ -22,10 +22,25 @@
         {% endif %}
 
         {% if request.user.is_apply_staff and report.can_submit %}
-            <form action="{% url "apply:projects:reports:skip" pk=report.pk %}" method="post">
-                {% csrf_token %}
-                <input type="submit" value="Skip" class="btn data-block__action-link"></input>
-            </form>
+
+            <input data-fancybox data-src="#skip-report-{{report.id}}" type="button" value="Skip" class="btn data-block__action-link"></input>
+
+            <!-- Skip report confirmation modal -->
+            <div class="modal" id="skip-report-{{report.id}}">
+                <h4 class="modal__header-bar modal__header-bar--no-bottom-space">Skip report</h4>
+                <div class="modal__copy">
+                    <p>You are about to skip the report for this period.</p>
```

Comment:
![image](https://user-images.githubusercontent.com/6338661/68197680-8ad53180-ffb2-11e9-9ef2-e267deea3e43.png)

Use `{{report.start_date}}` and `{{report.end_date}}`

Reply:
👍